subprojects/docs/src/docs/userguide/intro_multi_project_builds.adoc

```diff
@@ -31,11 +31,13 @@ Such builds come in all shapes and sizes, but they do have some common character
 
 
 The `settings.gradle` file tells Gradle how the project and subprojects are structured. Fortunately, you don’t have to read this file simply to learn what the project structure is as you can run the command `gradle projects`. Here's the output from using that command on the Java _multiproject_ build in the Gradle samples:
-++++
-<sample id="listProjects" dir="java/multiproject" title="Listing the projects in a build">
-                <output args="-q projects"/>
-            </sample>
-++++
+=== Example: Listing the projects in a build
```

Comment:
:x: Missing new-line before the heading:

![image](https://user-images.githubusercontent.com/132773/42990686-b4374dd4-8c03-11e8-86dd-97a3f507b2bd.png)